app/services/populate_topics.rb

```diff
@@ -5,7 +5,7 @@ def call
   end
 
   def topics_to_add
-    ox_topics - kafka_topics
+    TopicsConfig.new.uniq_topic_names - kafka_topics
   end
 
   def kafka_topics
```

Comment:
Can this be `existing_topics`?  All topics are "kafka" topics so that doesn't add to much.